forum/investors/models.py

```diff
@@ -1,3 +1,15 @@
 from django.db import models
+from users.models import CustomUser
+from django.utils import timezone
 
-# Create your models here.
+
+class Investor(models.Model):
+    investor_id = models.AutoField(primary_key=True)
+    user_id = models.ForeignKey(CustomUser, on_delete=models.CASCADE)
```

Comment:
It'll be better to rename `user_id` to `user` for clarity and consistency, as it is a foreign key to the `CustomUser` model.

Reply:
We store an integer in the user_id field. Django in many-to-many relation creates fields with _id. So it seems to me that changing user_id to user would create an inconsistency with Django's default behavior.

Reply:
By `investor_obj.user` we can access to `CustomUser` data.
In the table, this field will be named `user_id`.

Reply:
It's true, thank you!